gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Mar 29 15:52:32 IST 2018
+#Sun Feb 24 00:16:32 IST 2019
```

Comment:
Revert

Reply:
I have already created the issue 
https://github.com/fossasia/open-event-orga-app/issues/1514

Reply:
How does this solve that issue?

Reply:
After clicking on create button of step 3 nothing was happening. Now due to explicit intent, it will be forwarded to the eventDashboardPresenter class

Reply:
But does it create an event? The issue is that "Event name cannot be empty" error is showing on clicking the create button.